zip/src/main/kotlin/io/github/paulgriffith/kindling/zip/views/MultiToolView.kt

```diff
@@ -4,18 +4,30 @@ import com.formdev.flatlaf.extras.FlatSVGIcon
 import io.github.paulgriffith.kindling.core.MultiTool
 import io.github.paulgriffith.kindling.core.Tool
 import io.github.paulgriffith.kindling.core.ToolPanel
-import io.github.paulgriffith.kindling.zip.MultiPathView
 import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.spi.FileSystemProvider
 import javax.swing.JPopupMenu
 import kotlin.io.path.name
 import kotlin.io.path.outputStream
 
-class MultiToolView(override val provider: FileSystemProvider, override val paths: List<Path>) : MultiPathView() {
+class MultiToolView(
+    override val provider: FileSystemProvider,
+    override val paths: List<Path>,
+) : PathView("ins 0, fill") {
     private val multiTool: MultiTool
     private val toolPanel: ToolPanel
 
+    override val tabName by lazy {
+        val roots = paths.mapTo(mutableSetOf()) { path ->
+            path.name.trimEnd { it.isDigit() || it == '-' || it == '.' }
+        }
+        "[${paths.size}] ${roots.joinToString()}"
+    }
```

Comment:
A more useful tabname for multi-tools, hopefully; tries to represent the common part(s) of the path